Tests/SwiftPrettyPrintTests/Public/DebugTests.swift

```diff
@@ -114,7 +114,7 @@ class DebugTests: XCTestCase {
                              name: Optional("ポチ"),
                              nickname: nil,
                              age: 3,
-                             homepage: Optional(https://www.google.com/)
+                             homepage: Optional(URL("https://www.google.com/"))
```

Comment:
bugfix: `Optional<URL>` should print with type information when `debug` is true.

It's an old bug 😅